src/test/java/com/karankumar/bookproject/backend/entity/account/UserTest.java

```diff
@@ -61,7 +63,21 @@ void testValidUserSaved() {
                    .username("username")
                    .email("email");
     }
+    
+    @ParameterizedTest
+    @ValueSource(strings = {"123456789", "aPassWorD", "testPa$$123"})
+    void testPasswordIsInvalid(String input) {
```

Comment:
Your fork is quite far behind master. In the latest `UserTest` version on master, we're following a format where the display name is [class to test] 'should'. In this test class, the class has the display name `User should`. Each method then follows on from that sentence to complete a sentence (e.g. `notHaveAWeakPassword`). 

In line with our style guide, could you please rename your method to follow this format?